guides/v2.2/javascript-dev-guide/widgets/widget_dropdown.md

```diff
@@ -0,0 +1,202 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: Dropdown widget
+functional_areas: frontend,theme,dropdown
+---
+
+## Overview
+
+Widget source file is [lib/web/mage/dropdowns.js].
+
+The dropdown widget allows to show show on storefront a select box with custom
+content for each of available options.
+
+{:.bs-callout .bs-callout-info} Dropdown widget is not meant to replace default
+HTML select element. By replacing default select in store forms don't expect
```

Comment:
> By replacing default select in store forms don't expect the store to work 100% the same as before changes.

Could you provide some clarification there or, maybe, remove this section. Because currently it sounds mysterious and scary at the same time :)

Thank you.